test/parallel/test-module-parent-deprecation.js

```diff
@@ -0,0 +1,14 @@
+// Flags: --pending-deprecation
+
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+
+common.expectWarning(
+  'DeprecationWarning',
+  'module.parent is deprecated due to accuracy issues. Please use ' +
+          'require.main === module to check for CLI usage instead.',
```

Comment:
The mention of CLI here is a bit confusing.

Reply:
```suggestion
    'require.main to find program entry point instead.',
```